Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -2173,6 +2173,37 @@ void TipChangedHandler(object target, BlockChain<DumbAction>.TipChangedEventArgs
             // TODO: Add test cases for swap
         }
 
+        [Fact]
+        private async void Reorged()
+        {
+            Block<DumbAction> actualOldTip = default;
+            Block<DumbAction> actualNewTip = default;
+            Block<DumbAction> actualBranchpoint = default;
+
+            void ReorgedHandler(object target, BlockChain<DumbAction>.ReorgedEventArgs args)
+            {
+                actualOldTip = args.OldTip;
+                actualNewTip = args.NewTip;
+                actualBranchpoint = args.Branchpoint;
+            }
+
+            _blockChain.Reorged += ReorgedHandler;
+            var branchpoint = _blockChain.Tip;
+            var fork = _blockChain.Fork(_blockChain.Tip.Hash);
+            await fork.MineBlock(_fx.Address1);
+            await fork.MineBlock(_fx.Address2);
+            await _blockChain.MineBlock(_fx.Address3);
+
+            var oldTip = _blockChain.Tip;
+            var newTip = fork.Tip;
+
```

Comment:
Need to ensure if the event has never been triggered.

```suggestion
            Assert.Null(actualOldTip.Hash);
            Assert.Null(actualNewTip.Hash);
            Assert.Null(actualBranchpoint.Hash); 

```